.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: 'Checkout'
-        uses: actions/checkout@master
+        uses: actions/checkout@main
       - name: 'Terraform Format'
         uses: hashicorp/terraform-github-actions@master
```

Comment:
I can see this has been superseded by https://github.com/hashicorp/setup-terraform as well ✍️ perhaps some more cleanup in another task